src/background/index.ts

```diff
@@ -13,6 +13,13 @@ chrome.runtime.onInstalled.addListener(({reason}) => {
 
 chrome.runtime.setUninstallURL(UNINSTALL_URL);
 
+const welcome = `  /''''\\
```

Comment:
I think it can be reduced to 
```javascript
const welcome = `  /''''\\
 (0)==(0)
/__||||__\\
Welcome to Dark Reader!`;
console.log(welcome);

```

Reply:
I think I already added this in e6f5aa3

Reply:
It's not the same, I've narrowed down 
```
Welcome to Dark Reader!!!
`;
```
to
```
Welcome to Dark Reader!`;
```

Reply:
Ok, I will add that

Reply:
Added in 049b71a